### PR TITLE
Add tracetools package using source

### PR DESCRIPTION
**Pull request by iluetkeb**

### CODE REVIEW

**Comment by tfoote** [comment]
trying to retrigger travis

**Comment by iluetkeb** [comment]
@tfoote Is it normal that the check is taking so long?

**Comment by mikaelarguedas** [comment]
It looks like this has been flagged as an abuse by travis (https://travis-ci.org/ros/rosdistro/requests).

@tfoote I see that you "cleared" the account in #15650, can we do the same here ?

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -15062,6 +15062,21 @@ repositories:
...
     status: developed
+  tracetools:
+    doc:
+      type: git
+      url: https://github.com/bosch-robotics-cr/tracetools.git
+      version: devel
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/bosch-robotics-cr/tracetools.git
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: You should not use the same repository for the release.

Comment (tfoote) [reply indigo/distribution.yaml]: Yes, that will make a mess of your source repo.

**Comment by tfoote** [comment]
I've requested to clear the abuse flag with Travis Support

**Comment by iluetkeb** [comment]
All changes are flagged as an abuse by Travis CI. Does anybody have an idea what could cause that?

**Comment by mikaelarguedas** [comment]
It looks like your user and organization never logged into travis so they flagged it as abusive automatically. Is it possible for you to login into travis and then push an empty commit in here ? That should solve the issue.

Otherwise, can you contact travis support (support@travis-ci.com) to get your account/organization cleared ?

**Pull request closed without merging**